src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static int taskCounter = 0;
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static final int charLengthToSkip = 5;
+
+    // let statement be printed at center
+    public static void print(String str) {
```

Comment:
Consider changing the method name to make it more specific as to what it is printing, and to avoid any possible confusion with System.out.print()